packages/api/src/beacon/routes/debug.ts

```diff
@@ -29,6 +29,7 @@ const stringType = new StringType();
 const protoNodeSszType = new ContainerType(
   {
     executionPayloadBlockHash: stringType,
+    executionPayloadNumber: ssz.UintNum64,
```

Comment:
Is this API standardized?

Reply:
ummm ... i don't think so, double checked